library/test/tokens.test.ts

```diff
@@ -764,6 +765,7 @@ describe('Tokens', () => {
     it('should reject with status >= 400 <= 599', async () => {
       const parentId = chance.string();
       const tokenPayload = {
+        type: 'token' as TokenType,
```

Comment:
Do you face any errors without this cast?

Reply:
Yeah, I couldn't figure that out.. 
![image](https://user-images.githubusercontent.com/43392371/137966091-81b8fe56-1fa5-4be8-beef-43390dc9e09b.png)